tests/acceptance/features/apiActivities/activities.feature

```diff
@@ -2134,6 +2134,273 @@ Feature: check activities
       }
       """
 
+
+  Scenario: check activity with sort filter
+    Given user "Alice" has created folder "/New Folder"
+    And user "Alice" has uploaded file with content "ownCloud test text file 0" to "/New Folder/textfile.txt"
+    When user "Alice" lists the activities of folder "New Folder" from space "Personal" with sort "asc" using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["value"],
```

Comment:
The main thing here to check is the order of activites. This step doesn't do that.
So instead of `And the JSON data of the response should match`, maybe we can have a step like:
```feature
And the activities should be in the following order:
  | New Folder   | {user} added {resource} to {folder} |
  | textfile.txt | {user} added {resource} to {folder} |
```
Also, create activities so that similar activities doesn't occur one after another. This way we can know the order

Reply:
@saw-jan But i think the response is also as expected with respective sort value(asc / desc).

Reply:
the items order doesn't matter in the json schema. so we are not checking the order that way

Reply:
You can check if rearranging the items fails the test.

Reply:
> You can check if rearranging the items fails the test.

I just got to know it wont fail after rearranging the items.
So i will make a new step to check the order as you described above.